ext/standard/mail.c

```diff
@@ -320,7 +320,7 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
-	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+	if (hdr && strlen(hdr) && php_mail_detect_multiple_crlf(hdr)) {
```

Comment:
hmm, it's better to stash strlen(HDR) above while calculated it... then use here